proto/payload.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+import public "proto/utility.proto";
+
+package payload;
+
+enum PayloadType {
+    ACKNOWLEDGEMENT = 0;
+    REGISTRATION = 1;
+    DISCOVERY_REQUEST = 2;
+    DISCOVERY_RESPONSE = 3;
+    TASK_REQUEST = 4;
+    TASK_RESPONSE = 5;
+}
+
+message PayloadMessage {
+    string id = 1;
+    string senderId = 2;
+    bytes senderIpAddress = 3;
```

Comment:
Nit: is there a reason we use a byte array to represent this? We were previously just serializing/deserializing to string iirc

Reply:
Ah good catch. That should be the utility proto's IpAddress object. I'll update